### ISSUE TITLE
Pyright is not reporting error when calling a module

### ISSUE THREAD

**Issue by StockerMC** [microsoft/pyright#2694]
**Describe the bug**
Pyright is not reporting an error when a module is called, although it would produce an error at runtime.

**Expected behavior**
The following code to produce an error, consistent with mypy's behaviour.

**Screenshots or Code**
```python
import typing
typing()
```

**VS Code extension or command-line**
Are you running pyright as a VS Code extension or a command-line tool?
Command line (pyright 1.1.194)

**Additional context**
For the example code provided, mypy reports the following error:
```
error: Module not callable
```

**Comment by erictraut** [comment]
This will be fixed in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.195, which I just published. It will also be included in the next release of pylance.
